EngineeringModel.Tests/ViewModels/ElementDefinitionsBrowser/ElementDefinitionBrowserViewModelTestFixture.cs

```diff
@@ -819,5 +819,61 @@ public async Task Verify_that_ExecuteBatchChangeOwnershipElementDefinition_works
 
             this.changeOwnershipBatchService.Verify(x => x.Update(this.session.Object, It.IsAny<ElementDefinition>(), this.domain, true, It.IsAny<IEnumerable<ClassKind>>()), Times.Exactly(1));
         }
+
+        [Test]
+        public void VerifyThatExecuteSetAsTopeElementDefinitionCommandWorks()
+        {
+            var def2 = new ElementDefinition(Guid.NewGuid(), this.assembler.Cache, this.uri);
+            
+            this.iteration.Element.Add(this.elementDef);
+            this.iteration.Element.Add(def2);
+
+            this.iteration.TopElement = this.elementDef;
+
+            var vm = new ElementDefinitionsBrowserViewModel(this.iteration, this.session.Object, null, null, null, null, null, null);
+
+            var defRow = (ElementDefinitionRowViewModel)vm.ElementDefinitionRowViewModels.Single(x => x.Thing.Iid == this.elementDef.Iid);
```

Comment:
Probably a copy-paste thing, but please use `as ElementDefinitionRowViewModel` instead of direct type conversion.